crates/glaredb/src/args/mod.rs

```diff
@@ -49,6 +49,14 @@ pub struct RpcProxyArgs {
     /// Authorization code for communicating with Cloud.
     #[clap(long)]
     pub cloud_auth_code: String,
+
+    /// Path to server cert to use for TLS (required for TLS protocol).
+    #[clap(long)]
+    pub rpc_server_cert: Option<String>,
```

Comment:
If this is required to be a path, we should use `PathBuf` instead of `String`. 

To me, string implies that they could inline the cert `--rpc-server-cert=$MY_CERT` _(which we could add in a later PR if someone requests the functionality)_

I'm pretty sure clap will parse PathBuf's by default.